azure-iot-hub/azure/iot/hub/iothub_job_manager.py

```diff
@@ -0,0 +1,126 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .auth import ConnectionStringAuthentication
+from .protocol.iot_hub_gateway_service_ap_is import IotHubGatewayServiceAPIs as protocol_client
+
+
+class IoTHubJobManager(object):
+    """A class to provide convenience APIs for IoTHub Job Manager operations,
+    based on top of the auto generated IotHub REST APIs
+    """
+
+    def __init__(self, connection_string):
+        """Initializer for a Job Manager Service client.
+
+        After a successful creation the class has been authenticated with IoTHub and
+        it is ready to call the member APIs to communicate with IoTHub.
+
+        :param str connection_string: The IoTHub connection string used to authenticate connection
+            with IoTHub.
+
+        :returns: Instance of the IoTHubJobManager object.
+        :rtype: :class:`azure.iot.hub.IoTHubJobManager`
+        """
+
+        self.auth = ConnectionStringAuthentication(connection_string)
+        self.protocol = protocol_client(self.auth, "https://" + self.auth["HostName"])
+
+    def create_import_export_job(self, job_properties):
+        """Creates a new import/export job on an IoT hub.
+
+        :param job_properties job_properties: Specifies the job specification.
+
+        :raises: `HttpOperationError<msrest.exceptions.HttpOperationError>`
+            if the HTTP response status is not in [200].
+
+        :returns: JobProperties object containing the created job.
```

Comment:
>:returns: JobProperties object containing the created job. [](start = 8, length = 58)

You also will want to have a line below this like

:rtype: :class:`azure.iot.hub.models.JobProperties`